Evergreen.xcodeproj/project.pbxproj

```diff
@@ -646,7 +664,7 @@
 				849A97961ED9EFAA007D329B /* Extensions */,
 				849A97991ED9EFB6007D329B /* Resources */,
 				84FB9A2C1EDCD6A4003D53B9 /* Frameworks */,
-				849C64741ED37A5D003D8FC0 /* EvergreenTests */,
+				9451C1F51EE0778700605914 /* Tests */,
```

Comment:
EvergreenTests and the new EvergreenUITests were moved into a new Tests group for organization.